docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,58 @@
+# Prepare Amazon EKS cluster with GPU for KubeRay examples
+
+## Step 1: Create a Kubernetes cluster on Amazon EKS
+
+Follow the first two steps in this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#)
+to: (1) Create your Amazon EKS cluster (2) Configure your computer to communicate with your cluster.
+
+## Step 2: Create node groups for the Amazon EKS cluster
+
+You can follow "Step 3: Create nodes" in this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#) to create node groups. The following section provides more detailed information.
+
+### Create a CPU node group
+
+Typically, we do not run GPU workloads on the Ray head. Create a CPU node group for all Pods except Ray GPU 
```

Comment:
Could you give some more details on how large this node group should be? And could you specify what size node users should be using. You could do this by attaching a screenshot showing a working configuration from the AWS console.

Reply:
This document will be used for multiple examples, each may require a different setup. Take this PR as an example, I have provided additional details in Step 1 of [stable-diffusion-rayservice.md](https://github.com/ray-project/kuberay/pull/1181/files#diff-e2830c575dbdd61d5c4f45500ef2a56f0ef4777acccfb695a508d20dcd5224ca). 

(1) Do not provide details in [aws-eks-gpu-cluster.md](https://github.com/ray-project/kuberay/pull/1181/files/0eb5db1e2353f8ad9e38ef3a9b595cb350bf6ff5#diff-db316438b08e0a251e5c17308a77017d75ec589bdfaa716fef765aff981d3f8d). Instead, include the details in the example documentation. (current method)
(2) Provide a common configuration in this document, and if the example does not fit the configuration, include additional details in the example documentation.

Do you have any preference?

Reply:
I'd prefer (2). I don't want users start Ray with nodes that are too small and having it never start.

Reply:
Updated